packages/destination-actions/src/destinations/tiktok-audiences/removeFromAudience/index.ts

```diff
@@ -26,26 +27,36 @@ const action: ActionDefinition<Settings, Payload, AudienceSettings> = {
     enable_batching: { ...enable_batching },
     external_audience_id: { ...external_audience_id }
   },
-  perform: async (request, { audienceSettings, payload, statsContext }) => {
+  perform: async (request, { audienceSettings, payload, statsContext, features }) => {
+    // This flag hides mapping but we want to ensure the mappings are also not run.
+    if (features && !features[MIGRATION_FLAG_NAME]) {
+      return
+    }
+
     const statsClient = statsContext?.statsClient
     const statsTag = statsContext?.tags
 
     if (!audienceSettings) {
       throw new IntegrationError('Bad Request: no audienceSettings found.', 'INVALID_REQUEST_DATA', 400)
     }
 
-    statsClient?.incr('actions-tiktok-audiences.removeFromAudience', 1, statsTag)
+    statsClient?.incr('removeFromAudience', 1, statsTag)
     return processPayload(request, audienceSettings, [payload], 'delete')
   },
-  performBatch: async (request, { audienceSettings, payload, statsContext }) => {
+  performBatch: async (request, { audienceSettings, payload, statsContext, features }) => {
+    // This flag hides mapping but we want to ensure the mappings are also not run.
```

Comment:
NIT: I think the comment is redundant given that the code is self explanatory. I'd remove it from all actions.

```suggestion
    // This flag hides mapping but we want to ensure the mappings are also not run.
```